TombEngine/Objects/TR3/Entity/SophiaLeigh.cpp

```diff
@@ -245,33 +245,34 @@ namespace TEN::Entities::Creatures::TR3
 
 	static void SpawnSophiaLeighProjectileBolt(ItemInfo& item, ItemInfo* enemy, const CreatureBiteInfo& bite, SophiaData* data, bool isBoltLarge, short angleAdd)
 	{
-		int fxNumber = CreateNewEffect(item.RoomNumber);
+		int fxNumber = CreateNewEffect(item.RoomNumber, ID_PROJ_SHARD, item.Pose);
```

Comment:
Why object ID is reassigned immediately after `CreateNewEffect`? Here it is `ID_PROJ_SHARD`, later it becomes `ID_ENERGY_BUBBLES`.